src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ApplicationPropertyPage/ApplicationFrameworkValueProvider.cs

```diff
@@ -19,38 +33,72 @@ internal sealed class ApplicationFrameworkValueProvider : InterceptingPropertyVa
         private const string NoneItemType = "None";
         private const string ApplicationDefinitionItemType = "ApplicationDefinition";
 
+        internal const string ApplicationFrameworkProperty = "UseApplicationFramework";
+        internal const string EnableVisualStylesProperty = "EnableVisualStyles";
+        internal const string SingleInstanceProperty = "SingleInstance";
+        internal const string SaveMySettingsOnExitProperty = "SaveMySettingsOnExit";
+        internal const string HighDpiModeProperty = "HighDpiMode";
+        internal const string AuthenticationModeProperty = "VBAuthenticationMode";
+        internal const string ShutdownModeProperty = "ShutdownMode";
+        internal const string SplashScreenProperty = "SplashScreen";
+        internal const string MinimumSplashScreenDisplayTimeProperty = "MinimumSplashScreenDisplayTime";
+
         private readonly UnconfiguredProject _project;
         private readonly IProjectItemProvider _sourceItemsProvider;
+        private IMyAppFileAccessor _myAppXamlFileAccessor;
 
         [ImportingConstructor]
         public ApplicationFrameworkValueProvider(
             UnconfiguredProject project,
-            [Import(ExportContractNames.ProjectItemProviders.SourceFiles)] IProjectItemProvider sourceItemsProvider)
+            [Import(ExportContractNames.ProjectItemProviders.SourceFiles)] IProjectItemProvider sourceItemsProvider,
+            IMyAppFileAccessor MyAppXamlFileAccessor)
```

Comment:
nit: lower case first letter

```suggestion
            IMyAppFileAccessor myAppXamlFileAccessor)
```